paimon-core/src/main/java/org/apache/paimon/operation/FileStoreScan.java

```diff
@@ -106,11 +106,15 @@ default Long totalRecordCount(Snapshot snapshot) {
                 : snapshot.totalRecordCount();
     }
 
+    default List<SimpleFileEntry> readSimpleEntries() {
+        return readSimpleEntries(false);
+    }
+
     /**
      * Read {@link SimpleFileEntry}s, SimpleFileEntry only retains some critical information, so it
      * cannot perform filtering based on statistical information.
      */
-    List<SimpleFileEntry> readSimpleEntries();
+    List<SimpleFileEntry> readSimpleEntries(boolean containsTime);
```

Comment:
Can you introduce another method `readSimpleEntries(boolean containsTime)`? Keep old `readSimpleEntries()`.